index.js

```diff
@@ -72,17 +86,31 @@ var cssGlobbingPlugin = function(options) {
       files = [];
 
       if (globRegExp.exec(filePattern)) {
-        glob.sync(filePattern, { cwd: path.dirname(filename) }).forEach(function(foundFilename) {
-          if ((options.extensions.indexOf(path.extname(foundFilename)) !== -1)&&(options.ignoreFolders.indexOf(path.dirname(foundFilename))) == -1) {
-            files.push(foundFilename);
+        glob.sync(filePattern, { cwd: path.dirname(filename) }).forEach(function(foundFilePath) {
+          if ((options.extensions.indexOf(path.extname(foundFilePath)) !== -1)&&(options.ignoreFolders.indexOf(path.dirname(foundFilePath))) == -1) {
+
+            var foundFilename = foundFilePath.split('\\').pop().split('/').pop();
```

Comment:
I think we should defer more to `path` in this block – for example use `path.basename()` on this row, get the extension for replacement with `path.extname()`, join back together with `path.join()` + `path.dirname()`, etc.